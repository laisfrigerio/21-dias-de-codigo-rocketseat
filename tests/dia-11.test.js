const { JSDOM } = require('jsdom')
const fs = require('fs')
const path = require('path')

const html = fs.readFileSync(path.resolve(__dirname, '../dia-11/index.html'), 'utf8')

let dom
let container

describe('dia 11', () => {
  beforeEach(() => {
    dom = new JSDOM(html, { runScripts: 'dangerously' })
    container = dom.window.document.body
  })

  test('should match the login page challenge', () => {
    expect(container).toMatchSnapshot();
  });
})