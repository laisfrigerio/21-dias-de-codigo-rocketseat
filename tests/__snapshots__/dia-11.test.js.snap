// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dia 11 should match the login page challenge 1`] = `
<body
  class="main-container"
  id="particles-js"
>
  
  
  <header />
  
  
  <div
    class="glassmorphism-card"
  >
    
    
    <div
      class="card"
    >
      
      
      <a
        href="https://laisfrigerio.com.br/"
        title="Site/Porfolio da Lais"
      >
        
        
        <img
          alt="Avatar"
          class="avatar"
          src="https://avatars0.githubusercontent.com/u/20709086?s=460&u=13c9928ea02aa8349e70cac14ea211c40a8f80d9&v=4"
        />
        
      
      </a>
      
      
      <h1>
        Login
      </h1>
      
      
      <form
        action="#"
      >
        
        
        <div
          class="form-control"
        >
          
          
          <input
            name="email"
            placeholder="E-mail"
            type="email"
          />
          
        
        </div>
        
        
        <div
          class="form-control"
        >
          
          
          <input
            name="password"
            placeholder="Senha"
            type="password"
          />
          
        
        </div>
        
        
        <button
          type="button"
        >
          ACESSAR
        </button>
        
      
      </form>
      
    
    </div>
    
  
  </div>
  
  
  <label
    class="theme-mode"
  >
    
    
    <input
      id="slider"
      onchange="toggleTheme()"
      type="checkbox"
    />
    
    
    <span
      class="slider"
    />
    
  
  </label>
  
  
  <script
    src="../assets/js/particles.js"
  />
  
  
  <script
    src="../assets/js/theme.js"
  />
  
  
  <script
    src="../assets/js/include-html.js"
  />
  
  
  <script
    src="scripts.js"
  />
  
  
  <script
    src="https://cdn.jsdelivr.net/npm/particles.js@2.0.0/particles.min.js"
  />
  



</body>
`;
